Dockerfile.multi_arch

```diff
@@ -5,10 +5,15 @@
 # Keeping the old images available will allow older versions to be built
 # while allowing the new versions to make changes that are not backwards compatible.
 #
+# The base image here should be multi-arched beforehand when used on non-amd64 platforms, such as arm64
+# platform
```

Comment:
The same comment as left for the `Dockerfile.builder`, please keep the `Dockerfile` untouched and this file will be a new

Reply:
@aanm OK. I would keep the current Dockerfile and Dockerfile.builder unchanged.